### PR TITLE
PUB-2669 - CST and PHT Non Strategic Lists

### PR DESCRIPTION
**Pull request by ChrisS1512**
### JIRA link

https://tools.hmcts.net/jira/browse/PUB-2669

### Change description

Added Style Guide for CST and PHT Non Strategic Lists
The controller is generic, and should be able to be used for non strategic lists going forward.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/resources/locales/en/cst-and-pht-weekly-hearing-list.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "date": "Date",
+    "caseName": "Case name",
+    "hearingLength": "Hearing length",
+    "hearingType": "Hearing type",
+    "venue": "Venue",
+    "additionalInformation": "Additional Information",
```
Comment (KianKwa) [inline src/main/resources/locales/en/cst-and-pht-weekly-hearing-list.json:7]: Can we make the casing "Information" lower case to match up with the other header?

File: src/main/resources/locales/cy/cst-and-pht-weekly-hearing-list.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+    "date": "Dyddiad",
+    "caseName": "Enw’r Achos",
```
Comment (KianKwa) [inline src/main/resources/locales/cy/cst-and-pht-weekly-hearing-list.json:3]: Arhos needs to be in lower case

File: src/main/resources/locales/en/cst-and-pht-weekly-hearing-list.json (hunk 1 above)
Comment (ChrisS1512) [reply src/main/resources/locales/en/cst-and-pht-weekly-hearing-list.json]: Updated

File: src/main/resources/locales/cy/cst-and-pht-weekly-hearing-list.json (hunk 2 above)
Comment (ChrisS1512) [reply src/main/resources/locales/cy/cst-and-pht-weekly-hearing-list.json]: Updated

**Review by KianKwa — APPROVED** [review]

**Pull request closed without merging**